dpe/src/dpe_instance.rs

```diff
@@ -176,13 +176,12 @@ impl<C: Crypto> DpeInstance<'_, C> {
     /// Returns the number of TCIs written to `nodes`
     pub(crate) fn get_tcb_nodes(
         &self,
-        context: &Context,
+        start_idx: usize,
```

Comment:
nit: change the comment from `context` to `start_idx`

Reply:
Done